### PR TITLE
[OPIK-271] Add integration with aws bedrock

### PR DESCRIPTION
**Pull request by alexkuzmik**
## Details
Adds an integration to AWS Bedrock clients that works similarly to openai integration.

## Testing
Manual testing only was performed. Library integration test can be added only after we configure AWS credentials with access to bedrock in CI.

## Documentation
Added a docstring.

### CODE REVIEW

File: sdks/python/src/opik/api_objects/trace.py [hunk 1]
```diff
@@ -203,6 +204,13 @@ def log_feedback_score(
 class TraceData:
     """
     The TraceData object is returned when calling :func:`opik.opik_context.get_current_trace_data` from a tracked function.
+
```
Comment (jverre) [inline sdks/python/src/opik/api_objects/trace.py]: @alexkuzmik These get documented in the [public docs](https://www.comet.com/docs/opik/python-sdk-reference/Objects/TraceData.html), let's move these to a comment in the code itself

File: apps/opik-documentation/documentation/docs/cookbook/bedrock.ipynb [hunk 2]
```diff
@@ -0,0 +1,307 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using Opik with OpenAI\n",
```
Comment (jverre) [inline apps/opik-documentation/documentation/docs/cookbook/bedrock.ipynb]:
```suggestion
    "# Using Opik with AWS Bedrock\n",
```

File: apps/opik-documentation/documentation/docs/cookbook/bedrock.ipynb [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+    "\n",
+    "Opik integrates with OpenAI to provide a simple way to log traces for all OpenAI LLM calls. This works for all OpenAI models, including if you are using the streaming API.\n"
```
Comment (jverre) [inline apps/opik-documentation/documentation/docs/cookbook/bedrock.ipynb]:
```suggestion
    "Opik integrates with AWS Bedrock to provide a simple way to log traces for all Bedrock LLM calls. This works for all supported models, including if you are using the streaming API.\n"
```

File: apps/opik-documentation/documentation/docs/cookbook/bedrock.ipynb [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating an account on Comet.com\n",
+    "\n",
+    "[Comet](https://www.comet.com/site?from=llm&utm_source=opik&utm_medium=colab&utm_content=openai&utm_campaign=opik) provides a hosted version of the Opik platform, [simply create an account](https://www.comet.com/signup?from=llm&utm_source=opik&utm_medium=colab&utm_content=openai&utm_campaign=opik) and grab you API Key.\n",
```
Comment (jverre) [inline apps/opik-documentation/documentation/docs/cookbook/bedrock.ipynb]:
```suggestion
    "[Comet](https://www.comet.com/site?from=llm&utm_source=opik&utm_medium=colab&utm_content=bedrock&utm_campaign=opik) provides a hosted version of the Opik platform, [simply create an account](https://www.comet.com/signup?from=llm&utm_source=opik&utm_medium=colab&utm_content=bedrock&utm_campaign=opik) and grab you API Key.\n",
```

We don't really use the UTMs but worth updating them

File: apps/opik-documentation/documentation/docs/cookbook/bedrock.ipynb [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+    "\n",
+    "> You can also run the Opik platform locally, see the [installation guide](https://www.comet.com/docs/opik/self-host/overview/?from=llm&utm_source=opik&utm_medium=colab&utm_content=openai&utm_campaign=opik) for more information."
```
Comment (jverre) [inline apps/opik-documentation/documentation/docs/cookbook/bedrock.ipynb]:
```suggestion
    "> You can also run the Opik platform locally, see the [installation guide](https://www.comet.com/docs/opik/self-host/overview/?from=llm&utm_source=opik&utm_medium=colab&utm_content=bedrock&utm_campaign=opik) for more information."
```

File: apps/opik-documentation/documentation/docs/cookbook/bedrock.ipynb [hunk 6]
```diff
@@ -0,0 +1,307 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "from opik.integrations.bedrock import track_bedrock\n",
+    "\n",
+    "os.environ[\"OPIK_PROJECT_NAME\"] = \"bedrock-integration-demo\"\n",
+    "bedrock_client = track_bedrock(bedrock)"
```
Comment (jverre) [inline apps/opik-documentation/documentation/docs/cookbook/bedrock.ipynb]: You can pass the project name to the `track_bedrock` decorator, it's a bit cleaner than using the environment variable

File: apps/opik-documentation/documentation/docs/cookbook/bedrock.ipynb [hunk 7]
```diff
@@ -0,0 +1,307 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from opik import track\n",
+    "\n",
+    "os.environ[\"OPIK_PROJECT_NAME\"] = \"openai-integration-demo\"\n",
```
Comment (jverre) [inline apps/opik-documentation/documentation/docs/cookbook/bedrock.ipynb]: Better to pass the project name as part of the track decorator for the top most function

**Comment by alexkuzmik** [comment]
@japdubengsub ready for review

**Review by japdubengsub — APPROVED** [review]

**Pull request merged**